### PR TITLE
Bump version v1.7.0.

### PR DESCRIPTION
**Pull request by avas27JTG**
#### Summary
- https://github.com/mattermost/mattermost-plugin-zoom/pull/284
- https://github.com/mattermost/mattermost-plugin-zoom/pull/299
- https://github.com/mattermost/mattermost-plugin-zoom/pull/308
- https://github.com/mattermost/mattermost-plugin-zoom/pull/307
- https://github.com/mattermost/mattermost-plugin-zoom/pull/287
- https://github.com/mattermost/mattermost-plugin-zoom/pull/310
- https://github.com/mattermost/mattermost-plugin-zoom/issues/311
- https://github.com/mattermost/mattermost-plugin-zoom/issues/312
- https://github.com/mattermost/mattermost-plugin-zoom/issues/313
- https://github.com/mattermost/mattermost-plugin-zoom/pull/305
- https://github.com/mattermost/mattermost-plugin-zoom/pull/301
- https://github.com/mattermost/mattermost-plugin-zoom/pull/298
- https://github.com/mattermost/mattermost-plugin-zoom/pull/295
- https://github.com/mattermost/mattermost-plugin-zoom/pull/294
- https://github.com/mattermost/mattermost-plugin-zoom/pull/279
- https://github.com/mattermost/mattermost-plugin-zoom/pull/289
- https://github.com/mattermost/mattermost-plugin-zoom/pull/288
- https://github.com/mattermost/mattermost-plugin-zoom/pull/286
- https://github.com/mattermost/mattermost-plugin-zoom/pull/281
- https://github.com/mattermost/mattermost-plugin-zoom/pull/251
- https://github.com/mattermost/mattermost-plugin-zoom/pull/277
- https://github.com/mattermost/mattermost-plugin-zoom/pull/268
- https://github.com/mattermost/mattermost-plugin-zoom/pull/258
- https://github.com/mattermost/mattermost-plugin-zoom/pull/257
- https://github.com/mattermost/mattermost-plugin-zoom/pull/242
- https://github.com/mattermost/mattermost-plugin-zoom/pull/247
- https://github.com/mattermost/mattermost-plugin-zoom/pull/245
- https://github.com/mattermost/mattermost-plugin-zoom/pull/147
- https://github.com/mattermost/mattermost-plugin-zoom/pull/228
- https://github.com/mattermost/mattermost-plugin-zoom/pull/209
- https://github.com/mattermost/mattermost-plugin-zoom/pull/232

### CODE REVIEW

**Review by AayushChaudhary0001 — APPROVED** [review]
Tested and Approved
Working fine in all the conditions, LGTM.

**Comment by mickmister** [comment]
To be more clear, here are the definitions to use to follow semantic version, or "semver":

https://semver.org

Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

---

In our case this typically means:

- MAJOR - Breaking changes and (significant) deprecations
- MINOR - New features or significant change in how the program functions to the user. Basically a change in requirements of the program
- PATCH/DOT - Bug fixes

In this case of this release, it will probably be a minor release, since I think there are new features here, and probably not any intentional breaking changes

**Comment by avas27JTG** [comment]
Yeah I agree with you @mickmister the only reason we were thinking of making it a major release was due to removal deprecated JWT settings from the plugin https://github.com/mattermost/mattermost-plugin-zoom/pull/299

Please let me know if we should create the release as `1.7.0` or `2.0.0` considering above change?

**Comment by avas27JTG** [comment]
@mickmister I've updated the description after removing the issues/PRs which were released in previous releases.

**Comment by mickmister** [comment]
> the only reason we were thinking of making it a major release was due to removal of the deprecated JWT settings from the plugin https://github.com/mattermost/mattermost-plugin-zoom/pull/299

@avas27JTG That deprecation is really more of a deprecation from Zoom, and not so much our software. AFAIK the JWT offering doesn't work at all anymore on Zoom plugin version. Because of that, I don't think we should treat the new release as a breaking change in that regard. I think we should go with `1.7.0` for this release

**Comment by mickmister** [comment]
I removed some more from the list that are more clerical work, or have already been released. I think this may be more accurate:

- https://github.com/mattermost/mattermost-plugin-zoom/pull/284
- https://github.com/mattermost/mattermost-plugin-zoom/pull/299
- https://github.com/mattermost/mattermost-plugin-zoom/pull/308
- https://github.com/mattermost/mattermost-plugin-zoom/pull/307
- https://github.com/mattermost/mattermost-plugin-zoom/pull/287
- https://github.com/mattermost/mattermost-plugin-zoom/pull/310
- https://github.com/mattermost/mattermost-plugin-zoom/issues/311
- https://github.com/mattermost/mattermost-plugin-zoom/issues/312
- https://github.com/mattermost/mattermost-plugin-zoom/issues/313
- https://github.com/mattermost/mattermost-plugin-zoom/pull/305
- https://github.com/mattermost/mattermost-plugin-zoom/pull/251
- https://github.com/mattermost/mattermost-plugin-zoom/pull/257
- https://github.com/mattermost/mattermost-plugin-zoom/pull/147
- https://github.com/mattermost/mattermost-plugin-zoom/pull/228
- https://github.com/mattermost/mattermost-plugin-zoom/pull/209

**Comment by avas27JTG** [comment]
> > the only reason we were thinking of making it a major release was due to removal of the deprecated JWT settings from the plugin #299
> 
> @avas27JTG That deprecation is really more of a deprecation from Zoom, and not so much our software. AFAIK the JWT offering doesn't work at all anymore on Zoom plugin version. Because of that, I don't think we should treat the new release as a breaking change in that regard. I think we should go with `1.7.0` for this release

Sounds good, thanks for clarifying.

File: plugin.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "support_url": "https://github.com/mattermost/mattermost-plugin-zoom/issues",
     "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.5.1",
```
Comment (mickmister) [inline plugin.json:7]: FYI we always have to change this value too
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.7.0",
```

Comment (avas27JTG) [reply plugin.json]: Noted, thanks

**Comment by avas27JTG** [comment]
@mickmister created a new PR https://github.com/mattermost/mattermost-plugin-zoom/pull/322 as I also changed the branch name to `release-bump-1.7.0` in place of `...-2.0.0`.

**Pull request closed without merging**